tauv_vehicle/src/depth_sensor/depth_sensor.py

```diff
@@ -46,7 +46,7 @@ def start(self):
             depthmsg.header.stamp = rospy.Time.now()
             depthmsg.header.frame_id = "odom"
             depthmsg.depth = self.ms5837.depth()
-            self.pub_depth(depthmsg)
+            self.pub_depth.publish(depthmsg)
```

Comment:
this is a small unrelated fix that was causing the depth to not be published on the real hardware. oops!